### PR TITLE
Simplify and document onboarding

### PR DESCRIPTION
**Pull request by cesidio**
## Jira ticket URL
https://dfedigital.atlassian.net/browse/TEVA-1468

### CODE REVIEW

File: documentation/api-keys.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+In order to generate a new secret key:
+1. Run the `rails secret` task from the repo, it will generate a new secret key
+1. You need to generate a different key per environment
+1. Update `SECRET_KEY_BASE` in `set-*-govuk-paas-env.sh`
+
+### ROLLBAR_ACCESS_TOKEN
+Used to report server-side errors to Rollbar.
+1. Navigate to: Setting > Project access tokens
+1. Edit the token you want to revoke
+1. Select `Yes, disable this token` and click `Save
```
Comment (cpjmcquillan) [inline documentation/api-keys.md:38]: Typo on the `Save`

File: documentation/api-keys.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+1. Click `Create a new access token`
+1. Select `post_server_item` and click Save
```
Comment (cpjmcquillan) [inline documentation/api-keys.md:40]: `Save`

File: documentation/api-keys.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+### DFE_SIGN_IN_PASSWORD and DFE_SIGN_IN_SECRET
+* DFE_SIGN_IN_PASSWORD is used to encrypt JWT tokens to authorise a user with DFE sign-in.
+* DFE_SIGN_IN_SECRET is OAuth2 client secret. It is only know to the application and the authorising server.
+
+In order to update the password you need to have access to the DSI Manage console, it is specific to the environment (test, preprod, prod). This is the URL for test: https://test-manage.signin.education.gov.uk
+
+__Once you are inside:__
+1. Follow the `Service configuration` link
+1. Regenerate the `Client secret` (`DFE_SIGN_IN_SECRET` env varialbe) and `API secret` (`DFE_SIGN_IN_PASSWORD` env variable)
```
Comment (cpjmcquillan) [inline documentation/api-keys.md:58]: *variable

File: documentation/api-keys.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+1. Always use your own login if it has sufficient permissions.
+    * If it does not, request them on [#digital-tools-support](https://ukgovernmentdfe.slack.com/archives/CMS9V0JQL)
+    * Please only use teachingjobs@digital.education.gov.uk as a last resort.
+
+#### For a string-based API key follow this workflow:
+1. Click 'CREATE CREDENTIALS' in the toolbar at the top of the page
+1. Choose 'API key'
+1. Click 'RESTRICT KEY' in the modal that appears
+1. In 'Name' give the key a clear, descriptive name including environment details where applicable
+1. Under 'Application Restrictions' select 'HTTP referres (web sites)'
```
Comment (cpjmcquillan) [inline documentation/api-keys.md:79]: *referrers

**Review by cpjmcquillan — APPROVED** [review]

File: documentation/api-keys.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
+## API Keys
+
+### ALGOLIA_WRITE_API_KEY
+This key is used to index vacancies in Algolia.
+In order to manage API keys:
+1. Log in to Algolia using teachingjobs@digital.education.gov.uk
```
Comment (alexbowen) [inline documentation/api-keys.md:6]: should these not be numbered 1-x or be bullet points?

**Review by alexbowen — APPROVED** [review]

**Pull request merged**